docs/booking-post-mortem-2026-08-20.md

```
@@ -0,0 +1,212 @@
+| Clock | club +12ms, one-way 12ms, 104 probes, 5 transitions, **tick pinned to ±23ms** |
+| Lead | 24ms early |
+| Fire | Reserve **1/8** at **+1006ms** past the window (aim +1030, less the 24ms lead) |
+| Answer | **accepted** — club clock **+1001ms**, round trip **456ms**, 84077 bytes, 0 sheet rows, 0 Reserve buttons, `form=08:08 AM`, `popup=False` |
+| Ledger | `club granted 08:08 AM at +1006ms past the window` |
+| Outcome | `phase=complete, success=True, attempts=1, totalMs=1870` |
+| Verify | `RESERVATION_CHECK: Reservation found` |
+
+The `<eval>` on the accepted response is the accept signature and carries no
+`.show()`:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)